versioned_docs/version-2.0.0/concepts/about.md

```diff
@@ -4,7 +4,7 @@ title: What is Casper?
 
 ## What is Casper? {#what-is-casper}
 
-Casper is a [Turing-complete](../concepts/glossary/T.md#turing-complete-blockchain) smart-contracting platform, backed by a Proof-of-Stake (PoS) consensus algorithm and WebAssembly (Wasm). The network is a [permissionless](../concepts/glossary/P.md#permissionless), decentralized, public blockchain.
+Casper is a pseudo-[Turing-complete](../concepts/glossary/T.md#turing-complete-blockchain) smart-contracting platform, backed by a Proof-of-Stake (PoS) consensus algorithm and WebAssembly (Wasm). The network is a [permissionless](../concepts/glossary/P.md#permissionless), decentralized, public blockchain.
```

Comment:
Same question as above. Why/when did this change?

Reply:
Incorporated - changed it to "Casper is a [Turing-complete](https://github.com/casper-network/docs-redux/concepts/glossary/T.md#turing-complete-blockchain) (given the limitations of finite memory) smart-contracting platform"